cluster/examples/kubernetes/ceph/operator.yaml

```diff
@@ -54,6 +54,15 @@ spec:
         # (Optional) Rook Agent toleration key. Set this to the key of the taint you want to tolerate
         # - name: AGENT_TOLERATION_KEY
         #   value: "<KeyOfTheTaintToTolerate>"
+        # (Optional) Rook Agent NodeAffinity Label Key. 
+        # Set this to the key of the Node Label to match.
+        # - name: AGENT_NODE_LABEL_KEY
+        #   value: "<KeyOfTheLabelToMatch>"
+        # (Needed if AGENT_NODE_LABEL_KEY is set ) Rook Agent NodeAffinity Label Value.
+        # Set this to the value of the Node Label to match.(Comma separated values if multiple values 
```

Comment:
What if I have multiple labels I want to select on? It looks like I can only select on multiple values, but not multiple labels. Would that be useful?

Reply:
Agree. We need to be able to select multiple labels each with multiple values. Will add that.